### max number of send_topics supported: 50

### list all <IPname: IP> here (IPname can be set arbitrarily) ###
IP:
  self: '*'   # '*' stands for all self IPs
  usv:   192.168.147.211
  suav:  192.168.147.21
  tuav1: 192.168.147.120
  tuav2: 192.168.147.41
  tuav3: 192.168.147.51
  local: 127.0.0.1 # just for local machine test

####### Send these ROS messages to remote robots #######
## if no send_topics needed, comment all these out
send_topics: 
- topic_name: /tuav2_takeoff_flag
  msg_type: std_msgs/Int8
  max_freq: 1
  srcIP: self
  srcPort: 3101

- topic_name: /tuav3_takeoff_flag
  msg_type: std_msgs/Int8
  max_freq: 1
  srcIP: self
  srcPort: 3102



####### receive these ROS messages from remote robots #######
## if no recv_topics needed, comment all these out
recv_topics:
- topic_name: /usv_localization_imu
  msg_type: sensor_msgs/Imu 
  srcIP: usv 
  srcPort: 1101 

- topic_name: /tuav1_takeoff_flag
  msg_type: std_msgs/Int8
  srcIP: usv 
  srcPort: 1102

